src/error.rs

```diff
@@ -20,11 +20,11 @@ pub enum Error {
         size: usize,
         msg: &'static str,
     },
-    #[cfg(feature = "std")]
     /// A custom Scroll error for reporting messages to clients
-    Custom(String),
     #[cfg(feature = "std")]
+    Custom(String),
     /// Returned when IO based errors are encountered
+    #[cfg(feature = "std")]
```

Comment:
i just realized this got added; this is technically a breaking change, but nostd could never have worked with this, iiuc?

Reply:
correct, `io` does not exist without `std`, so technically it is not a breaking change :)